server/src/infra/entities/asset-stack.entity.ts

```diff
@@ -1,7 +1,7 @@
 import { Column, Entity, JoinColumn, OneToMany, OneToOne, PrimaryGeneratedColumn } from 'typeorm';
 import { AssetEntity } from './asset.entity';
 
-@Entity('asset_stack')
+@Entity('asset_stack', { synchronize: false })
```

Comment:
Why did you have to add this?

Reply:
It tries to drop the stack index, but also doesn't work if I put an index decorator because `assetsId` and `tagsId` don't exist on this entity.

Reply:
Oh, I confused this with the tag relation. It actually works without that so I think I just needed to have the index in the asset entity instead.